code/modules/reagents/chemistry/reagents/toxins.dm

```diff
@@ -502,6 +502,19 @@
 	M.adjustToxLoss(1)
 	..()
 
+/datum/reagent/somnistimine
+	name = "Somnistimine"
+	id = "somnistimine"
+	description = "A powerful tranquilizer."
+	reagent_state = LIQUID
+	color = "#2E2E61"
+	metabolization_rate = 0.4
+
+/datum/reagent/somnistimine/on_mob_life(mob/living/M)
+	if(current_cycle>= 3)
```

Comment:
6 seconds is long enough for a security officer to tase you. 20 seconds is so long that they can disposal themselves, run through a door, or otherwise escape easily. Plus... everything Bigtimetripod said.